CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [2.0.2]
+* Modified mock_canvas copy from the framework to unblock move to flutter_test
+
 ## [2.0.1]
 * Modified mock_canvas copy from the framework to unblock move to flutter_test
```

Comment:
Since both entries are identical, do we need to update the version? If so, let's link to the relevant PR in each entry.

Reply:
I am fine either way, was just following the instructions:

```html
- [x] I have incremented the package version as appropriate and updated
    `CHANGELOG.md` with my changes <!-- THIS IS REQUIRED -->
```

I'm ok not incrementing it if you are! :)

Reply:
Touchè!! I'm fine with leaving it at 2.0.1.